pandas/io/json/normalize.py

```diff
@@ -41,6 +44,11 @@ def nested_to_record(ds, prefix="", sep=".", level=0):
 
     level: the number of levels in the jason string, optional, default: 0
 
+    max_level: normalize to a maximum level of, optional, default: None
```

Comment:
Did you by chance already look at the `_recursive_extract` function in `json_normalize`? It has a `level` parameter already.

Haven't stepped through the code in detail to see but am curious if there's not a better way of going about this then adding a `max_level` parameter here; it's not very intuitive what the difference between `level` and `max_level` would be with these functions

Reply:
Will it be fine if I change it to `start_level` and `end_level`?

Reply:
Not sure that's the right solution; my general concern with this PR is that it doesn't make the API that much cleaner for these funcs.

Can you add some tests that interact with the already existing keywords? Maybe I'm just not thinking about it correctly. That would help from an understanding perspective as well as ensuring better regression testing